package.json

```
@@ -37,8 +37,8 @@
     "access": "public"
   },
   "sideEffects": false,
-  "types": "./dist/types/index.d.ts",
-  "main": "./dist/cjs/index.js",
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
```

suggestion(strong): please define a module path as well.

```suggestion
  "main": "./dist/index.js",
  "module": "./dist/index.mjs",
  "types": "./dist/index.d.ts",
```